### DISCUSSION TITLE
About Dynamic Recompiler

### DISCUSSION THREAD

**Discussion by ShanZeHu** [ptitSeb/box64#841] [Q&A]
Hello, can BOX64 disable the dynamic recompile module and just use the interpreted execution to execute the translated program? If so, how do I disable this dynamic recompile feature?

**Comment by ptitSeb** [comment]
You can use `BOX64_DYNAREC=0` env.var. to disable Dynamic recompiler.

**Reply by ShanZeHu** [reply]
Okay, thank you very much for your answer
